src/neo/Cryptography/MPT/MPTProofStore.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using Neo.Persistence;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.Cryptography.MPT
+{
+    public class MPTProofStore : IReadOnlyStore
```

Comment:
Where is it used?

Reply:
https://github.com/neo-project/neo/blob/master/src/neo/Persistence/MemoryStore.cs

I think this can be replaced by `MemoryStore`. What do you think? @KickSeason